pkgs/applications/misc/far2l/default.nix

```diff
@@ -1,29 +1,23 @@
 { lib, stdenv, fetchFromGitHub, fetchpatch, makeWrapper, cmake, pkg-config, wxGTK30, glib, pcre, m4, bash,
-  xdg-utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, darwin }:
+  xdg-utils, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz, imagemagick, darwin,
+  libuchardet, spdlog, xercesc, fmt, openssl, libssh, samba, neon, libnfs, libarchive }:
 
-let
-  newer-colorer-schemes = fetchFromGitHub {
-    owner = "colorer";
-    repo = "Colorer-schemes";
-    rev = "7c831f5e94a90530ace8b2bb9916210e3a2fcda6"; # 2019-11-28 (far2l has older Colorer-schemes)
-    sha256 = "18vaahdz5i7xdf00c9h9kjjswm4jszywm8zkhva4c4ivr4qqnv2c";
-  };
-in
 stdenv.mkDerivation rec {
   pname = "far2l";
-  version = "2019-12-14.git${builtins.substring 0 7 src.rev}";
+  version = "2020-12-30.git${builtins.substring 0 7 src.rev}";
 
   src = fetchFromGitHub {
     owner = "elfmz";
     repo = "far2l";
-    rev = "dceaa3918ea2c5e43600bad3fc63f861b8d26fc4";
-    sha256 = "1ssd3hwz4b7vl4r858d9whl61cn23pgcamcjmvfa6ysf4x2b7sgi";
+    rev = "52c1372441443aafd1a7dff6f17969a6ec19885d";
+    sha256 = "0s7427fgxzj5zkyy6mhb4y5hqa6adsr30m0bigycp12b0495ryx0";
   };
 
   nativeBuildInputs = [ cmake pkg-config m4 makeWrapper imagemagick ];
 
-  buildInputs = [ wxGTK30 glib pcre ]
-    ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Cocoa;
+  buildInputs = [ wxGTK30 glib pcre libuchardet spdlog xercesc fmt ] # base requirements of the build
+    ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Cocoa # some macintosh stuff
```

Comment:
```suggestion
    ++ lib.optional stdenv.isDarwin Cocoa
```

Reply:
Are you 100% sure of this change? I mean, I haven't written this and I don't have a Mac to check it. I merely formatted the lines when adding new optional packages (to make more built-in archiver plugins work correctly) with a few comments on what is what. I can make this change, just a bit afraid to break something for people out there on Mac without even knowing. Just to confirm.

Reply:
> Are you 100% sure of this change?

Yes, this is a no-op change and just some cleanup. I can build it afterwards to check if it is broken.